components/Button.tsx

```diff
@@ -3,7 +3,7 @@ interface Props {
   color: "blue" | "white" | "orange";
   size?: "small";
   icon?: React.ReactNode;
-  onClick?: () => void;
+  onClick?: (e: React.MouseEvent<HTMLButtonElement>) => void;
```

Comment:
Hvorfor denne?

Reply:
Den gjør at man submite f.eks: (e: BaseSyntheticEvent) => {
                    submit(e);
                  } I send inn knappen for Committee-Interview-Times.tsx. Skal ikke påvirke de andre onClick funksjonene